tools/dev/run_dev_server.sh

```diff
@@ -45,6 +45,10 @@ case $CONFIG in
       ENABLE_MODULES="text2vec-contextionary,backup-filesystem" \
       CLUSTER_GOSSIP_BIND_PORT="7100" \
       CLUSTER_DATA_BIND_PORT="7101" \
+      RAFT_PORT="8300" \
+      INTRA_RPC_PORT="8301" \
+      BOOTSTRAP_EXPECT="3" \
```

Comment:
I would suggest to have a common prefix for RAFT settings, so I would suggest to rename those variables to:
```
INTRA_RPC_PORT -> RAFT_RPC_PORT
BOOTSTRAP_EXPECT -> RAFT_BOOTSTRAP_EXPECT
```

Reply:
and what means `INTRA_RPC_PORT`? as meaning `Inter communication RPC port`?

Reply:
ohhh, I looked further to the bottom and saw the description:
```
the port used for internal RPCs within the cluster
```
can we change the name to a more descriptive like: `RAFT_RPC_PORT` or `RAFT_INTERNAL_RPC_PORT`?

Reply:
I agree it would be clearer, I'll change it